src/components/Edit/ChangeSummary.vue

```diff
@@ -43,25 +43,44 @@
       </div>
     </template>
 
+    <!-- Office Addresses -->
     <template v-if="officeAddressesChanged">
-      <v-divider class="mx-4" />
+      <v-divider class="mx-8" />
       <div id="address-summary-section" class="section-container">
         <OfficeAddresses :isSummaryView="true" />
       </div>
     </template>
+
+    <!-- Org Persons -->
+    <template v-if="hasOrgPersonChanged">
+      <v-divider class="mx-8" />
+      <div id="org-person-summary-section" class="section-container">
+        <v-row no-gutters>
+          <v-col cols="12" sm="3">
+            <label><strong>Proprietor Information</strong></label>
```

Comment:
Labels may already have bold styles, and if not then they should :)

Reply:
They do indeed, thank you, will fix!

Reply:
Fixed